src/extension/background-script/connectors/nwc.ts

```diff
@@ -215,6 +218,15 @@ class NWCConnector implements Connector {
   connectPeer(args: ConnectPeerArgs): Promise<ConnectPeerResponse> {
     throw new Error("Method not implemented.");
   }
+
+  private convertCustomRecords(
```

Comment:
we have 2 methods now `convertCustomRecords` and `mapTLVRecords`

Can they be renamed so it's obvious in what direction we are doing mapping?

Reply:
Done!